solutions/standard/variables.tf

```
@@ -68,26 +68,20 @@ variable "secret_manager_tags" {
 
 variable "skip_kms_iam_authorization_policy" {
   type        = bool
-  description = "Set to true to skip the creation of an IAM authorization policy that permits all Secrets Manager instances in the resource group to read the encryption key from the KMS instance. If set to false, pass in a value for the KMS instance in the existing_kms_instance_guid variable."
-  default     = false
+  description = "A flag to create an IAM authorization that grants Secrets Manager access to a Key Protect resource(true) or not(false). Set to false if a preexisting authorization can be used."
+  default     = true
 }
 
 variable "existing_sm_kms_key_crn" {
   type        = string
-  description = "The CRN of an existing KMS key to use for Secrets Manager. If not supplied, a new key ring and key will be created."
+  description = "The CRN of an existing KMS key to use for Secrets Manager. If not supplied, a new key ring and new key will be created."
```

You don't need `new` in either object. You can't create something "old".

```suggestion
  description = "The CRN of a KMS key to use for Secrets Manager. If not specified, a key ring and key are created."
```